pandas/tests/groupby/test_size.py

```diff
@@ -18,6 +18,29 @@ def test_size(df, by):
         assert result[key] == len(group)
 
 
+@pytest.mark.parametrize(
+    "by",
+    [
+        [0, 0, 0, 0],
+        [0, 1, 1, 1],
+        [1, 0, 1, 1],
+        [0, None, None, None],
+        pytest.param([None, None, None, None], marks=pytest.mark.xfail),
```

Comment:
@rhshadrach im trying to chase down xfails and im not even sure what this means.  what do you expect `df.groupby(by=[None]*4, ...)` to do?

Reply:
Current the `[0, 1, 1, 1]` case is returning 

```
0    1
1    3
dtype: int64
```

When `dropna=True`, I'd expect `4 * [None]` to return `Series([], dtype='int64')` and when `dropna=False`, I'd expected

```
np.nan   1
dtype: int64
```